src/python/zquantum/core/wip/circuit/conversions2/cirq_conversions.py

```diff
@@ -135,6 +139,21 @@ def _rotation(angle: Parameter) -> cirq.EigenGate:
 qubit_index = attrgetter("x")
 
 
+@overload
+def export_to_cirq(gate: Union[_gates.Gate]) -> cirq.Gate:
```

Comment:
The union here isn't needed, right?
```suggestion
def export_to_cirq(gate: _gates.Gate) -> cirq.Gate:
```